src/pages/client/RegisterPage/RegisterPage.jsx

```diff
@@ -1,98 +1,115 @@
-                                label="Nombre"
-                                name="fullName"
-                                placeholder="Elon Musk"
-                            />
-                        </div>
-                        <div className={styles.formControl}>
-                            <TextInput
-                                className={styles.input}
-                                label="Correo"
-                                name="email"
-                                placeholder="elon@tesla.com"
-                            />
-                        </div>
-                        <div className={styles.formControl}>
-                            <TextInput
-                                className={styles.input}
-                                label="DNI"
-                                name="dni"
-                                placeholder="47898661"
-                            />
-                        </div>
-                        <div className={styles.formControl}>
-                            <TextInput
-                                className={styles.input}
-                                label="Dirección"
-                                name="address"
-                                placeholder="Av. Los Tilos 167, Surco"
-                            />
-                        </div>
-                        <div className={styles.formControl}>
-                            <TextInput
-                                className={styles.input}
-                                label="Contraseña"
-                                type="password"
-                                name="password"
-                                placeholder="ElonMusk1"
-                            />
-                        </div>
-                        <Button className={styles.button} name="Submit" type="submit" />
-                        <div
-                            style={{
-                                textAlign: 'center',
-                            }}>
-                            <p>¿No tienes una cuenta? Regístrate aquí</p>
-                            <p>¿Eres colaborador? Click aquí</p>
-                        </div>
-                    </Form>
-                )}
-            </Formik>
-        </PageWrapper>
+        <Layout className="flex flex-col md:flex-row">
+            <AuthHeader loginTheme={false}>
+                <Logo className="w-32 h-auto md:mb-2 md:fill-tertiary" />
+                <h1 className="text-2xl font-bold w-100 self-center hidden md:block">
+                    Encuentra personas de confianza para organizar tu hogar
+                </h1>
+            </AuthHeader>
+            <section className="flex flex-col h-full w-full md:min-w-[450px] md:overflow-auto">
+                <main className="flex justify-center h-full w-full items-baseline">
+                    <div className="flex flex-col justify-center w-full max-w-xl px-12 pt-8">
```

Comment:
I'd rather you not use `<div>` for different sections of your components. Not a necessary change, but please think about creating custom sections in the future